livefs_edit/actions.py

```diff
@@ -333,7 +333,7 @@ def add_autoinstall_config(ctxt, autoinstall_config):
 
 
 @register_action()
-def resign_pool(ctxt):
+def resign_pool(ctxt, dist='stable'):
```

Comment:
I think it would be better to default dist to None and do `if dist is None: dist = ctxt.get_suite()`

Reply:
I set `dist=None` [here](https://github.com/mwhudson/livefs-editor/pull/19/commits/971c140580253de53d0b136e27d2f6927ad7357d).
I think the `if` is not needed, because of this assignment:

https://github.com/mwhudson/livefs-editor/blob/971c140580253de53d0b136e27d2f6927ad7357d/livefs_edit/actions.py#L403-L405